pegasus/forms/professional_development_workshop.rb

```diff
@@ -119,9 +119,12 @@ def self.solr_query(params)
       first_date_dt: '[NOW TO *]',
     }.map{|key,value| "#{key}:#{value}"}.join(' AND ')
 
+    fl = 'location_p,location_name_p,location_name_s,dates_ss,id'
```

Comment:
I looked at which fields were referenced in the js callbacks:

This form is used in [workshop-search.js](https://github.com/code-dot-org/code-dot-org/blob/3a51f3f515906fae664aef47dc16b557d9cfaee5/pegasus/sites.v3/code.org/public/js/workshop-search.js#L25-L89)

And the ClassSubmission form (pegasus/forms/class_submission.rb) is used in class-search.js [here](https://github.com/code-dot-org/code-dot-org/blob/8cb63298fcf6626864d056a2ee199c92b23e1ae7/pegasus/sites.v3/code.org/public/js/class-search.js#L93-L122) and [here](https://github.com/code-dot-org/code-dot-org/blob/8cb63298fcf6626864d056a2ee199c92b23e1ae7/pegasus/sites.v3/code.org/public/js/class-search.js#L194-L240).